ambry-api/src/main/java/com/github/ambry/account/Container.java

```diff
@@ -84,6 +84,7 @@
   static final boolean CACHEABLE_DEFAULT_VALUE = true;
   static final Set<String> CONTENT_TYPE_WHITELIST_FOR_FILENAMES_ON_DOWNLOAD_DEFAULT_VALUE = Collections.emptySet();
 
+  public static final short CONTAINER_ID_START_RANGE = 8;
```

Comment:
This can be confusing to someone who is only aware of OS ambry.

Reply:
True, make this as a config